docs/source/examples/combine_techniques.md

```diff
@@ -0,0 +1,107 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Composing techniques: Readout Error Mitigation and Zero Noise Extrapolation
+
+Noise in quantum computers can arise from a variety of sources, and sometimes applying multiple error mitigation techniques can be more beneficial than applying a single technique alone. Here we apply a combination of readout error mitigation (REM) and zero noise extrapolation (ZNE) to a randomized benchmarking (RB) task. In REM, the inverse transition / confusion matrices are generated and applied to the noisy measurement results. In ZNE, the expectation value of the observable of interest is computed at different noise levels, and subsequently the ideal expectation value is inferred by extrapolating the measured results to the zero-noise limit. More information on the [REM](../guide/rem.md) and [ZNE](../guide/zne.md) techniques can be found in the corresponding sections of the user guide.
+
++++
+
+## Setup
+
+We begin by importing the relevant modules and libraries required for the rest of this tutorial.
+
+```{code-cell} ipython3
+import cirq
+import numpy as np
+from mitiq.benchmarks import generate_rb_circuits
+from mitiq import MeasurementResult, Observable, PauliString, raw
+```
+
+## Task
+
+We will demonstrate using REM + ZNE on RB circuits, which are generated using Mitiq's built-in benchmarking function, `generate_rb_circuits`. More information on the RB protocol is available [here](https://learn.qiskit.org/course/quantum-hardware/randomized-benchmarking). In this example we use a two-qubit RB circuit with a Clifford depth (number of Clifford groups) of 5.
+
+```{code-cell} ipython3
+circuit = generate_rb_circuits(2, 5)[0]
+```
+
+## Noise model and executor
+
+The noise in this example is a combination of depolarizing and readout errors, the latter of which are modeled as bit flips immediately prior to measurement. We use an [`Executor`](../guide/executors.md) function to execute the quantum circuit with the noise model applied.
+
+```{code-cell} ipython3
+def execute(circuit: cirq.Circuit, noise_level: float = 0.005, p0: float = 0.05) -> MeasurementResult:
+    """Execute a circuit and return a ``MeasurementResult``, with depolarizing noise of strength
+    ``noise_level`` and readout errors with probability ``p0``."""
```

Comment:
These few lines should be formatted so they don't overflow the text box:
<img width="691" alt="Screenshot 2023-05-04 at 11 11 19 PM" src="https://user-images.githubusercontent.com/12703123/236388143-fce46a69-3635-4cb7-9a9f-610718e2af9b.png">

One method to do this is copy and paste the code into a mitiq codefile and let `black` format it.

Also, the double ticks are not needed since it's being formatted as code. Maybe not even single backticks?

Finally, I think it flows a little better like this. If you want to mention the return type, probably best to add a new sentence.
```suggestion
def execute(circuit: cirq.Circuit, noise_level: float = 0.005, p0: float = 0.05) -> MeasurementResult:
    """Execute a circuit with depolarizing noise of strength ... and readout errors ...
```